hardware/Schematic-Design-Guide.md

```diff
@@ -26,42 +33,58 @@ There are relative reference designs for all of the solutions in the website. Ba
+<div align="center">
+  <img src="files\HW-Schematic-Design-Guide\Proprietary-Page.png">  
+</div> 
+
+In this page, you can move mouse to "Products" tab, then the pop-down menu will appear with Wireless products which include the Proprietary item, then click left button of your mouse. And it will enter the Proprietary product list [web page](https://www.Silabs.com/wireless/proprietary) as following:
+<div align="center">
+  <img src="files\HW-Schematic-Design-Guide\Proprietary-Product.png">  
+</div>
+
+You can find the generations of proprietary products as listed in the bottom table. You can select "EFR32FG1 series 1 sub-GHz and 2.4GHz SoCs" for more [details](https://www.Silabs.com/wireless/proprietary/efr32fg1-series-1-sub-ghz-2-4-ghz-socs):
+<div align="center">
+  <img src="files\HW-Schematic-Design-Guide\Proprietary-S1.png">  
+</div>
 
-![](files/HW-Schematic-Design-Guide/Proprietary-Page.png)
-In this page, you can move mouse to Products tab, then the pop-down menu will appear with Wireless products which include the Proprietary item, then click left button of your mouse. And it will enter the Proprietary product list page as following:
-![](files/HW-Schematic-Design-Guide/Proprietary-Product.png)
-You can find the generations of proprietary products as listed in the bottom table. You can select EFR32FG1 series 1 sub-GHz and 2.4GHz SoCs for more details:
-![](files/HW-Schematic-Design-Guide/Proprietary-S1.png)
-In this page, you can find the brief introduction of the proprietary series 1 chips and the parameter list of each chips. Ignore the chips list, you can click the View Document in this page and acess the documents list pages directly:
-![](files/HW-Schematic-Design-Guide/Proprietary-Reference.png)
-For example, If you want to look for 434MHz +10dBm output power reference design, you can click item EFR32FG1 BRD4251B 2400/433 MHz 10dBm Radio Board Full Design Package and download the reference design package to your local address.
-The schematic likes:[BRD4251B](https://www.silabs.com/documents/public/schematic-files/EFR32FG1-BRD4251B-B00-schematic.pdf)
+In this page, you can find the brief introduction of the proprietary series 1 chips and the parameter list of each chips. Ignore the chips list, you can click the "View Document" in this page and acess the documents list [pages](https://www.Silabs.com/support/resources.p-wireless_proprietary_efr32fg1-series-1-sub-ghz-and-2-4-ghz-socs) directly:
+<div align="center">
+  <img src="files\HW-Schematic-Design-Guide\Proprietary-Reference.png">  
+</div>
+
+For example, If you want to look for 434MHz +10dBm output power reference design, you can click item "EFR32FG1 BRD4251B 2400/433 MHz 10dBm Radio Board Full Design Package" and download the reference design package to your local address.
+The schematic likes: [BRD4251B](https://www.Silabs.com/documents/public/schematic-files/EFR32FG1-BRD4251B-B00-schematic.pdf)
```

Comment:
Should change to "The BRD4251B schematic for example can be found here."

Reply:
Corrected.